src/caselawclient/models/press_summaries.py

```diff
@@ -1,6 +1,54 @@
+from functools import cached_property
+from typing import Any
+
+from ds_caselaw_utils import neutral_url
+
+from caselawclient.xml_helpers import get_xpath_match_string
+
 from .documents import Document
 
 
 class PressSummary(Document):
     document_noun = "press summary"
     document_noun_plural = "press summaries"
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        self.attributes_to_validate = self.attributes_to_validate + [
+            (
+                "has_ncn",
+                True,
+                "This {document_noun} has no neutral citation number",
+            ),
+            (
+                "has_valid_ncn",
+                True,
+                "The neutral citation number of this {document_noun} is not valid",
+            ),
+        ]
+
+        super(PressSummary, self).__init__(*args, **kwargs)
+
+    @cached_property
+    def neutral_citation(self) -> str:
+        return get_xpath_match_string(
+            self.content_as_xml_tree,
+            "/akn:akomaNtoso/akn:doc/akn:mainBody/akn:p/akn:neutralCitation/text()",
```

Comment:
everything apart from the xpath line, i.e where we find the neutral citation in the xml is the same between this and the Judgment class.

I am contemplating moving most of the code to the Document class and just setting the xpath on each subclass. Then if we introduce new document types which we want to extend Document class from we can change things if needed

Reply:
have refactored so now have a NeutralCitationMixin class as per @jacksonj04 's suggestion on slack.